src/bp/core/app/botpress.ts

```diff
@@ -183,6 +183,13 @@ export class Botpress {
   private async maybeStartLocalSTAN() {
     const config = await this.moduleLoader.configReader.getGlobal('nlu')
 
+    if (!process.LOADED_MODULES['nlu']) {
```

Comment:
I didn't know about this variable `LOADED_MODULES`

Reply:
Same! It's by working with the config-reader that I learned about it!